bundles/org.eclipse.platform.doc.user/whatsNew/platform_whatsnew.html

```diff
@@ -21,10 +21,10 @@
   ul {padding-left: 13px;}
   /*]]>*/
   </style>
-  <title>Eclipse Platform What's New in 4.25</title>
+  <title>Eclipse Platform What's New in 4.27</title>
```

Comment:
pom and manifest version update required

Reply:
https://github.com/eclipse-platform/eclipse.platform.common/blob/184b0084227904dd2ddd5d71d13afd298358f875/bundles/org.eclipse.jdt.doc.isv/pom.xml#L20-L23

it does not have a version section; looks like it inherits the parent's one?